flink-connectors/flink-connector-base/pom.xml

```diff
@@ -37,7 +37,7 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-core</artifactId>
+			<artifactId>flink-streaming-java</artifactId>
```

Comment:
Should we move `MultipleFuturesAvailabilityHelper` to `flink-core` rather than changing the dependencies of this module in a rather significant way? @zentol WDYT?

Reply:
+1, I think it makes sense. There's potential for more for usages in future connectors to help with future management, no pun intended.

The only dependencies of this class are in `flink-core` and the classes that depend on `MultipleFuturesAvailabilityHelper` are only in `flink-streaming-java`, and now `flink-connectors-base`.